src/platform-connection/platform-connection.service.ts

```diff
@@ -34,7 +36,12 @@ export class PlatformConnectionService {
     }
 
   async create(createSocialCredentialDto: CreatePlatformConnectionDto) {
-    return await this.repository.addPlatformConnection(createSocialCredentialDto);
+    const retVal = await this.repository.addPlatformConnection(createSocialCredentialDto);
+    this.queueService.addUserIdToQueue(
```

Comment:
man need to add the parameters that has been used here...

https://github.com/EeroHeikkinen/earthpoints-poc/pull/86